docassemble/juvenilesealing/data/questions/shared-content.yml

```diff
@@ -47,17 +47,6 @@ content: |
   These reports will give a summary of all your criminal charges
   in Massachusetts and tell you how each case ended.
 ---
-id: records_can_help_text
```

Comment:
Are we sure we want to remove this twice? We're removing it in both `shared-content` and in `sealing-body`. I think that means this completely disappears from the codebase.

Reply:
We had this in *three* places. It now exists only once.